### DISCUSSION TITLE
New valve game "Deadlock" optimization

### DISCUSSION THREAD

**Discussion by DarthJiggles** [Whisky-App/Whisky#1101] [Q&A]
Has anyone figured out how to get deadlock running fairly well without any freezes or crashing. 

currently running an apple m2 pro chip and am freezing/crashing multiple times per game.

**Comment by josephjoel64** [comment]
After weeks of testing me and my friends finally figured out what causes the endgame freezing. It's the buff/debuff spam that most endgame builds end up at, seems like Whisky can't handle the character overlays.
